release/apparmor.go

```diff
@@ -184,10 +184,25 @@ func MockAppArmorFeatures(kernelFeatures, parserFeatures []string) (restore func
 // probe related code
 
 var (
+	// requiredAppArmorParserFeatures denotes the features that must be present in the parser.
+	// Absence of any of those features results in the effective level be at most UnusableAppArmor.
 	requiredAppArmorParserFeatures = []string{
 		"unsafe",
 	}
+	// preferredAppArmorParserFeatures denotes the features that should be present in the parser.
+	// Absence of any of those features results in the effective level be at most PartialAppArmor.
+	preferredAppArmorParserFeatures = []string{
+		"unsafe",
+	}
+	// requiredAppArmorKernelFeatures denotes the features that must be present in the kernel.
+	// Absence of any of those features results in the effective level be at most UnusableAppArmor.
 	requiredAppArmorKernelFeatures = []string{
+		// TODO: what is the minimal set?
+		"file",
```

Comment:
I think you can remove the TODO and say: `// For now, require at least file and simply prefer the rest`